File: Hafta 6/pset6/cash.py
from cs50 import get_float

change = 0
count = 0

# While change is NAN, prompt user
while change <= 0:
    change = get_float("Change: ")
    cents = round(change * 100)


while cents > 0:
    if cents >= 25:
        cents = cents - 25
        count = count + 1
    elif cents >= 10:
        cents = cents - 10
        count = count + 1
    elif cents >= 5:
        cents = cents - 5
        count = count + 1
    elif cents >= 1:
        cents = cents - 1
        count = count + 1

print(count)

File: Hafta 1/problem setleri/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>
int main(void)
{
    float change;
    do
    {
        change = get_float("How much change is owned:");
    }
    while (change < 0);
    int cents = round(change * 100);
    int coin = 0;
    while (cents >= 25)
    {
        cents -= 25;
        coin++;
    }
    while (cents >= 10)
    {
        cents -= 10;
        coin++;
    }
    while (cents >= 5)
    {
        cents -= 5;
        coin++;
    }
    while (cents >= 1 && cents > 0)
    {
        cents -= 1;
        coin++;
    }
    printf("%i\n", coin);  
}